lib/rubocop/cop/layout/space_before_comma.rb

```diff
@@ -4,6 +4,13 @@ module RuboCop
   module Cop
     module Layout
       # Checks for comma (,) preceded by space.
+      #
+      # @example
+      #   # bad
+      #   [1 , 2 , 3]
+      #
+      #   # good
+      #   [1, 2, 3]
```

Comment:
This cop also detects comma of method/block arguments.
For example:

```ruby
a(1 , 2)
each { |a , b| }
```
So could you add these to good/bad examples?